fifth-project.html

```diff
@@ -0,0 +1,194 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>fifth_project</title>
+	<style>
+		* {
+			box-sizing:border-box;
+			}
+		body {
+			font-family:Segoe, "Segoe UI", "DejaVu Sans", "Trebuchet MS", Verdana, "sans-serif";
+			max-width: 1200px;
+			margin: auto;
+			background-color: aliceblue;
+			}
+		h1 {
+			margin: auto;
+			font-size: 1.6em;
+			letter-spacing: 5px;
+			}
+		h2 {
+			text-transform: none;
+			text-align: center;
+			}
+		p {
+			width: 80%;
+			margin: auto;
+			font-size: 1.2em;
+			line-height: 1.4;
+			text-indent: 20px;
+			text-align: left;
+			}
+		section {
+			margin-top: 20px;
+			border: 1px solid lightgray;}
+			}
+		
+		
+		/* This is the code for the project blocks */
+		div.gallery {
+			margin: 5px;
+    		border: 1px solid #ccc;
+    		float: left;
+    		width: 180px;
+			}
+		div.gallery:hover {
+			border: 1px solid #777;
+			opacity: .5;
+			filter: alpha(opacity=50); /* For IE8 and earlier */
+			}
+		div.gallery img {
+			width: 100%;
+			height: auto;
+			}
+		div.desc {
+    		padding: 15px;
+    		text-align: center;
+			}
+		.project-tile {
+			display: inline;
+			align-content: center;
+			}
+		
+		
+		/* This is the code for the navbar  */
+		#navbar {
+			position: fixed;
+			top:0;
+			display: inline-flex;
+			width: 100%;
+			background-color: aliceblue;
+			height: 70px;
+			padding-top: 20px;
+			padding-bottom: 20px;
+			align-content:center;
+		margin-left: 0px;}
+		ul {
+			list-style-type: none;
+			text-transform: uppercase;
+			margin: auto;
+			}
+		li {
+			display: inline-block;
+			}
+		li a {
+			text-decoration: none;
+			color: black;
+			background-color:bisque;
+			padding: 10px;
+			border: 2px solid bisque;
+			border-radius: 10px;
+			}
+		li a:hover {
+			background-color:cornsilk;
+			}
+		
+		
+		#welcome-section {
+			display: block;
+			text-align: center;
+			height: 100%;
+			margin-top: 70px;
+			padding: 50px;
+			}
+		#projects-section {
+			}
+		#contact-section {}
+		
+		@media screen and (max-width: 480px) {	
+			navbar {
```

Comment:
this doesn't affect the navbar when the screen is less than 480. Idk why.
I want each button to have its own block.